example-rules.yml

```diff
@@ -2,13 +2,13 @@ groups:
 - name: example-node-exporter-rules
   rules:
   # The count of CPUs per node, useful for getting CPU time as a percent of total.
-  - record: instance:node_cpus:count
-    expr: count(node_cpu{mode="idle"}) without (cpu,mode)
+  - record: instance:node_cpu_seconds_total:count
```

Comment:
The name doesn't match the metric, the name says is this the number of modes multiplies by the number of cpus.